tests/test_replication.py

```diff
@@ -3,11 +3,13 @@
 from mock import Mock
 from sydent.threepid import ThreepidAssociation
 from sydent.threepid.signer import Signer
-from tests.utils import make_request, make_sydent
+from tests.utils import make_request, make_sydent, setup_logging
 from twisted.web.client import Response
 from twisted.internet import defer
 from twisted.trial import unittest
 
+setup_logging()
```

Comment:
doing this in every test file seems a bit redundant (I'm surprised it even works tbh, but ok)

I'd suggest just calling it once in a file that is `import`ed by every test file (like, say, `tests.utils`).